docs/source/membership/membership.md

```diff
@@ -172,11 +172,11 @@ A local MSP folder contains the following sub-folders:
 
   **Note:** If you are using a [Hardware Security Module(HSM)](../hsm.html) for key management, this folder is empty because the private key is generated by and stored in the HSM.
 
-* **signcert:** For a peer or orderer node (or in a client's local MSP) this folder contains the node's **signing key**. This key matches cryptographically the node's identity included in **Node Identity** folder and is used to sign data --- for example to sign a transaction proposal response, as part of the endorsement phase.
+* **signcert:**  For a peer or orderer node (or in a client’s local MSP) this folder contains the node’s certificate issued by CA. This certificate contains the node's **public key**. While private key is used to digitally sign a transaction, public key is used to verify that the signature attached to the transaction is valid.
```

Comment:
Is used to sign a transaction proposal response for example as in the original explanation, not just a blanket 'transaction'.

Reply:
I went through the original explanation of the signature of the transaction proposal response(https://hyperledger-fabric.readthedocs.io/en/latest/peers/peers.html?highlight=transaction proposal response#phase-1-proposal). It says "A peer endorses a proposal response by adding its digital signature, and signing the entire payload using its private key." As per my understanding(reference: Digital Signature definition at https://github.com/bitcoinbook/bitcoinbook/blob/develop/ch06.asciidoc), although digital signature uses both private and public keys, essentially in this scenario, private key is always used to sign and public key is always used to verify the signature. 
So, when the original text says "This key..and is used to sign data --- for example to sign a transaction proposal response, as part of the endorsement phase.", it seems incorrect since signcert folder contains public key and is meant to verify the signature and not sign the data.

Please let me know if my understanding is incorrect.

Reply:
Yes, private keys are used to sign, and public keys are used to verify the signature, but I fear we are being a bit too specific here.

From a client, there is a signature over the proposal, there is a signature over the transaction. From the peer, there is a signature over the proposal response (the endorsement), as well as over requests to the orderer to retrieve blocks.  From the orderer there are signatures over the blocks themselves.

I also think the emphasis here should be that the certificate, which does embed a public key, is attached to node actions, like creating a transaction, or an endorsement, or a block, and it is this embedded certificate which is used to verify the signature.

Reply:
Agree with you. Thanks.